README.md

```diff
@@ -88,12 +88,153 @@ kubectl patch clusteradmissionpolicy psp-capabilities -p '{"metadata":{"finalize
-release. It follows the [SPDX](https://spdx.dev/) version 2.2 format and you can
-find it together with the signature and certificate used to sign it in the
-[release assets](https://github.com/kubewarden/kubewarden-controller/releases)
+```shell
+cosign verify-blob --certificate-oidc-issuer=https://token.actions.githubusercontent.com  \
+    --certificate-identity-regexp="https://github.com/kubewarden/kubewarden-controller*"  \
+    --bundle kubewarden-controller-attestation-amd64-provenance-cosign.bundle \
+    kubewarden-controller-attestation-amd64-provenance.json
+```
+
+To verify verify the attestation manifest and its layers signatures:
+
+```shell
+cosign verify --certificate-oidc-issuer=https://token.actions.githubusercontent.com  \
+    --certificate-identity-regexp="https://github.com/kubewarden/kubewarden-controller*" \
+    ghcr.io/kubewarden/kubewarden-controller@sha256:1abc0944378d9f3ee2963123fe84d045248d320d76325f4c2d4eb201304d4c4e
+```
+
+That sha256 hash is the digest of the attestation manifest or its layers.
+Therefore, you need to find this hash in the registry using the UI or tools
+like `crane`. For example, the following command will show you all the
+attestation manifests of the `latest` tag:
+
+```shell
+crane manifest  ghcr.io/kubewarden/kubewarden-controller:latest | jq '.manifests[] | select(.annotations["vnd.docker.reference.type"]=="attestation-manifest")'
+{
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "digest": "sha256:fc01fa6c82cffeffd23b737c7e6b153357d1e499295818dad0c7d207f64e6ee8",
+  "size": 1655,
+  "annotations": {
+    "vnd.docker.reference.digest": "sha256:611d499ec9a26034463f09fa4af4efe2856086252d233b38e3fc31b0b982d369",
+    "vnd.docker.reference.type": "attestation-manifest"
+  },
+  "platform": {
+    "architecture": "unknown",
+    "os": "unknown"
+  }
+}
+{
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "digest": "sha256:e0cd736c2241407114256e09a4cdeef55eb81dcd374c5785c4e5c9362a0088a2",
+  "size": 1655,
+  "annotations": {
+    "vnd.docker.reference.digest": "sha256:03e5db83a25ea2ac498cf81226ab8db8eb53a74a2c9102e4a1da922d5f68b70f",
+    "vnd.docker.reference.type": "attestation-manifest"
+  },
+  "platform": {
+    "architecture": "unknown",
+    "os": "unknown"
+  }
+}
+```
+
+Then you can use the `digest` field to verify the attestation manifest and its
+layers signatures.
+
+```shell
+cosign verify --certificate-oidc-issuer=https://token.actions.githubusercontent.com  \
+    --certificate-identity-regexp="https://github.com/kubewarden/kubewarden-controller*" \
+    ghcr.io/kubewarden/kubewarden-controller@sha256:fc01fa6c82cffeffd23b737c7e6b153357d1e499295818dad0c7d207f64e6ee8
+
+crane manifest  ghcr.io/kubewarden/kubewarden-controller@sha256:fc01fa6c82cffeffd23b737c7e6b153357d1e499295818dad0c7d207f64e6ee8                                                                                                  ✔ │ 14:28:50
```

Comment:
Ops, let's remove the time from my zsh theme. xD

Reply:
Woops, done! :sweat_smile: